### ISSUE TITLE
Cannot running with error no named parameter with name 'alignment'

### ISSUE THREAD

**Issue by MrHafid** [koukibadr/Elegant-Notification#70]
<img width="1264" alt="Screen Shot 2022-10-12 at 15 50 02" src="https://user-images.githubusercontent.com/43329305/195296898-47c4b676-41df-4416-a568-4d886c9f6510.png">

I have a problem with this and many other snackbar / Toast packages.
When i run the project, I have error in image.
Doe anyone have a fix for this?

**Comment by MrHafid** [comment]
@koukibadr I use package Flutter 2.2.3, in SDK 2.12.0, elegant_notification 1.6.4 maybe that ?

**Comment by koukibadr** [comment]
can you try with an older version of the package example 1.6.3 
I will check this error in the new version update

**Comment by MrHafid** [comment]
@koukibadr  Yah, I use the version 1.6.3 and after update package i have that error

**Comment by koukibadr** [comment]
Okay @MrHafid  I will check the new update so

**Comment by MrHafid** [comment]
@koukibadr no i not using attrbute that, I use the default setting

**Comment by koukibadr** [comment]
@MrHafid  Okay I will check it and let you know what's the problem exactly

**Comment by MrHafid** [comment]
@koukibadr what version your flutter ?, i check error ```
alignment
```  in widget AlertDialog maybe I have a version flutter not same with your package, cause alignment in my AlertDialog not found attribute that

**Comment by koukibadr** [comment]
Yes that's the issue but the package needs to be compatible with all version of flutter

**Comment by MrHafid** [comment]
@koukibadr ahhh i see, okay i will wait for update, maybe fix the issue. Thanks so much

**Comment by MrHafid** [comment]
hey @koukibadr I got a solution for that problem, before I asked permission to change the package, now the solution I made was by changing the AlertDialog with Material, like this.

<img width="425" alt="Screen Shot 2022-10-18 at 11 06 48" src="https://user-images.githubusercontent.com/43329305/196333329-0c9f9076-d708-40d7-9b0a-3044fb5dbfb4.png">

<img width="437" alt="Screen Shot 2022-10-18 at 11 10 37" src="https://user-images.githubusercontent.com/43329305/196333629-74b81125-2b66-4d2c-aa53-6e8c0720a349.png">
Write the code same the first function.

<img width="347" alt="Screen Shot 2022-10-18 at 11 10 44" src="https://user-images.githubusercontent.com/43329305/196333649-9ccd0841-64de-4426-916e-ffdaf016bd23.png">
result
